misc/netkitten/netkitten.go

```diff
@@ -44,11 +44,6 @@ func main() {
 	for {
 		// If given a target, work with it first
 		var cconn net.Conn
-		defer func() {
-			if cconn != nil {
-				cconn.Close()
-			}
-		}()
```

Comment:
Can `cconn` declaration and defer function be moved out of for() as another implementation? That should still make sure that we have a single call to defer

Reply:
Reusing the same `cconn` singleton between iterations results in `broken pipe` errors because the listener closes its end of the connection after the first write.